amazon_msk/datadog_checks/amazon_msk/data/conf.yaml.example

```diff
@@ -24,60 +35,48 @@ instances:
     #
     # node_exporter_port: 11002
 
-    ## @param tags - list of key:value elements - optional
-    ## List of tags to attach to every metric, event and service check emitted by this integration.
-    ## Learn more about tagging: https://docs.datadoghq.com/tagging/
+    ## @param prometheus_url - string - required
+    ## The URL where your application metrics are exposed by Prometheus.
     #
-    # tags:
-    #   - <KEY_1>:<VALUE_1>
-    #   - <KEY_2>:<VALUE_2>
+    prometheus_url: <PROMETHEUS_URL>
 
-    ## @param prometheus_timeout - integer - optional - default: 10
-    ## Overrides the default timeout value in second
+    ## @param prometheus_metrics_prefix - string - optional
+    ## <PREFIX> for exposed Prometheus metrics.
     #
-    # prometheus_timeout: 10
+    # prometheus_metrics_prefix: <PREFIX>_
 
     ## @param health_service_check - boolean - optional - default: true
-    ## Send a service check reporting about the health of the prometheus endpoint
-    ## It's named aws.msk.prometheus.health
+    ## Send a service check reporting about the health of the Prometheus endpoint.
+    ## The service check is named <NAMESPACE>.prometheus.health
     #
     # health_service_check: true
 
-    ## @param metrics - list of strings - optional
```

Comment:
I don't see this used in code but is not part of the openmetrics template either

Reply:
Yup only used by custom checks

Reply:
What do you mean? If our code is not reading this param then custom checks have to add its logic around it, right?
My question is if is safe to remove this param